src/main/java/com/redhat/management/approval/InputParser.java

```diff
@@ -33,57 +33,37 @@ public static Request parseRequest(java.util.LinkedHashMap<String, Object> reque
     email=bcd@123.com, first_name=bcd, last_name=efg}]}
     */
 
-    public static java.util.ArrayList<Group> parseGroups(java.util.ArrayList<java.util.LinkedHashMap<String, Object>> rawGroups) {
-        java.util.ArrayList<Group> groups = new java.util.ArrayList<Group>();
-        for (java.util.LinkedHashMap<String, Object> rawGroup : rawGroups) {
-            java.util.ArrayList<java.util.LinkedHashMap<String, String>> rawApprovers = (java.util.ArrayList<java.util.LinkedHashMap<String, String>>) rawGroup.get("users");
-            java.util.ArrayList<Approver> approvers = new java.util.ArrayList<Approver>();
+    public static ArrayList<Group> parseGroups(ArrayList<LinkedHashMap<String, Object>> rawGroups) {
```

Comment:
can it be
`public static List<Group> parseGroups(List<Map<String, Object>> rawGroups){`